tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -1150,6 +1283,10 @@ def create_custom_host_from_nodes(nodes, node_roles,
         else:
             config = rke_config_windows
 
+    elif NETWORK_PLUGIN == "calico":
+        config = rke_config_flatcar_calico
+    elif NETWORK_PLUGIN == "canal":
+        config = rke_config_flatcar_canal
```

Comment:
Should be done only in the case of flatcar os

Reply:
Changed this logic